prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -12,6 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+resource "google_monitoring_alert_policy" "pod-crashlooping" {
+  for_each     = var.prow_components
+  project      = var.project
+  display_name = "pod-crashlooping-${each.key}"
+  combiner     = "OR" # required
+
+  conditions {
+    display_name = "pod-crashlooping-${each.key}"
+
+    condition_monitoring_query_language {
+      # Alert if the service crashlooped, which results in more than 5 restarts in 5 minutes.
+      # This threshold is higher than the default crashloop backoff threshold, mainly due to
+      # the fact prow components would crashloop when kubernetes master is upgrading, which
+      # normally takes 5 minutes. Setting this to 10 minutes for excluding this case
+      duration = "1200s"
```

Comment:
This is 20mins not 12mins. It might be clearer to use `m` units rather than `s` if that is supported. The comment also needs to be updated.

Reply:
unfortunately `m` is not supported, updated the rest